src/util/syscall_sandbox.cpp

```diff
@@ -823,7 +821,9 @@ bool SetupSyscallSandbox(bool log_syscall_violation_before_terminating)
             return false;
         }
     }
-    SetSyscallSandboxPolicy(SyscallSandboxPolicy::INITIALIZATION);
+    if (use_for_main_thread) {
```

Comment:
nit: Could move this entire code block to the caller site instead of making it a bool arg.

Reply:
Done